tests/lib/rules/no-missing-require.js

```diff
@@ -43,11 +43,6 @@ ruleTester.run("no-missing-require", rule, {
             code: "require('eslint');",
             env: { node: true },
         },
-        {
-            filename: fixture("test.js"),
-            code: "require('eslint/lib/api');",
-            env: { node: true },
-        },
```

Comment:
This test was not valid because `eslint/lib/api` is not referenced in the eslint exports field (https://github.com/eslint/eslint/blob/main/package.json#L10-L14).

Reply:
maybe change it to `require("eslint/use-at-your-own-risk");`?

Reply:
mmm, that's a good test for export mapping